### ISSUE TITLE
Create megamenu's website equivalent

### ISSUE THREAD

**Issue by hellcp** [openSUSE/chameleon#39]
In case we can't embed dom and script, it would be useful to be able to link to a website containing all of those links meta-o-o or menu-o-o sound like viable names

**Comment by guoyunhe** [comment]
Yeah, I have thought this before. We have a lot of links that cannot be added to the megamenu because of limited space. I created a repo for this purpose https://github.com/openSUSE/universe-o-o
